public/components/register_model/error_call_out.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EuiCallOut, EuiText } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import { useFormContext } from 'react-hook-form';
+import { FORM_ERRORS } from './constants';
+
+import type { ModelFileFormData, ModelUrlFormData } from './register_model.types';
+
+export const ErrorCallOut = () => {
+  const form = useFormContext<ModelFileFormData | ModelUrlFormData>();
+
+  const errors = useMemo(() => {
+    const messages: string[] = [];
+    Object.keys(form.formState.errors).forEach((errorField) => {
+      const error = form.formState.errors[errorField as keyof typeof form.formState.errors];
+      const errorMessage = FORM_ERRORS.find(
+        (e) => e.field === errorField && e.type === error?.type
```

Comment:
Will we have multiple errors for one field?

Reply:
Good question, it may have, I will consult UX for this

Reply:
I think this question won't block this PR. Add a TODO is enough. We will change it after UX confirmed.

Reply:
Actually I just noticed that the configuration object has multiple errors, I'd assume we need to display all the errors